cms/static/js/xblock/authoring.js

```diff
@@ -34,50 +32,44 @@
                 }
             });
 
-            // Verification partitions (user can select exactly one)
-            if (element.find('#verification-access-checkbox').prop('checked')) {
-                partitionId = parseInt($('#verification-access-dropdown').val(), 10);
-                groupAccess[partitionId] = [ALLOW_GROUP_ID];
-            }
-
             return groupAccess;
         };
 
-        // When selecting "all students and staff", uncheck the specific groups
-        element.find('.field-visibility-level input').change(function(event) {
```

Comment:
Look @efischer19, I found more verification user partition changes I can delete! I'm removing some of the Studio code as part of my current work because I stumbled across it, and some of it I would need to modify if it stays in.